packages/ra-ui-materialui/src/button/PrevNextButtons.stories.tsx

```diff
@@ -0,0 +1,366 @@
+                        <Show
+                            actions={
+                                <MyTopToolbar>
+                                    <PrevNextButtons linkType="show" />
+                                    <EditButton />
+                                </MyTopToolbar>
+                            }
+                        >
+                            <DefaultSimpleShowLayout />
+                        </Show>
+                    }
+                />
+            </AdminUI>
+        </AdminContext>
+    </MemoryRouter>
+);
+
+export const WithLimit = () => (
+    <MemoryRouter>
+        <AdminContext dataProvider={dataProvider} i18nProvider={i18nProvider}>
+            <AdminUI>
+                <Resource
+                    name="customers"
+                    list={
+                        <List>
+                            <DefaultDataGrid />
+                        </List>
+                    }
+                    edit={
+                        <Edit
+                            redirect={false}
+                            actions={
+                                <MyTopToolbar>
+                                    <PrevNextButtons limit={500} />
+                                    <ShowButton />
+                                </MyTopToolbar>
+                            }
+                        >
+                            <DefaultSimpleForm />
+                        </Edit>
+                    }
+                    show={
+                        <Show
+                            actions={
+                                <MyTopToolbar>
+                                    <PrevNextButtons
+                                        linkType="show"
+                                        limit={500}
+                                    />
+                                    <ShowButton />
+                                </MyTopToolbar>
+                            }
+                        >
+                            <DefaultSimpleShowLayout />
+                        </Show>
+                    }
+                />
+            </AdminUI>
+        </AdminContext>
+    </MemoryRouter>
+);
+
+export const WithStyle = () => (
+    <AdminContext dataProvider={dataProvider}>
+        <ResourceContext.Provider value="customers">
+            <RecordContextProvider value={data.customers[0]}>
+                <PrevNextButtons
+                    sx={{
+                        color: 'blue',
+                    }}
+                />
+                <PrevNextButtons
+                    linkType="show"
+                    sx={{
+                        '& .RaPrevNextButton-list': {
+                            marginBottom: '20px',
+                            color: 'red',
+                        },
+                    }}
+                />
+            </RecordContextProvider>
+        </ResourceContext.Provider>
+    </AdminContext>
+);
+
+export const ErrorState = () => (
```

Comment:
We also need a story to test the loading state, with a getList returning a Promise that never resolves.